project/src/main/java/com/google/autograder/data/Database.java

```diff
@@ -167,23 +167,9 @@ public static void addQuestion(String questionName, String questionType, int que
       questionEntity.setProperty("points", questionPoints);
       questionEntity.setProperty("assignmentKey", assignmentKey);
       save(questionEntity);
+      System.out.println("question " + questionName + " has been added");
```

Comment:
Looks good. You can remove this though.

Reply:
oh yeah I just have this here because otherwise there's no confirmation right now that the question has been added. Just for my own sanity haha